src/components/elements/table/tableRowActionUtil.tsx

```diff
@@ -11,8 +11,7 @@ export const BASE_ACTION_COLUMN_DEF: ColumnDef<any> = {
   key: 'Actions',
   tableCellProps: {
     sx: {
-      py: 0,
-      px: 0,
+      p: 1,
```

Comment:
Slightly unrelated, but I think this `BASE_ACTION_COLUMN_DEF` needs a comment to explain usage. iiuc it should only be used in certain cases where we _can't_ use the default table linking behavior (by passing `rowLinkTo` OR `handleRowClick` OR `rowSecondaryActionConfigs`). For example it's used by the Bulk Services table to control rendering of the table row action menu so it can inject the assign service button. Is that fair?